tests/metrics.yaml

```diff
@@ -66,3 +66,40 @@ CollectorUp:
       - 'vrops_collector_up{collector="HostSystemPropertiesCollector",metric="vrops_hostsystem_properties"}'
       - 'vrops_collector_up{collector="DatastoreStatsCollector",metric="vrops_datastore_stats"}'
       - 'vrops_collector_up{collector="CollectorUp",metric="vrops_collector_up"}'
+
+VMStatsCollector:
+   metrics:
+      - 'vrops_vms_stats{cluster="cluster2",datacenter="datacenter2",hostsystem="hostsystem2",statkey="number_of_vcpus_total",virtualmachine="vm2"}'
+      - 'vrops_vms_stats{cluster="cluster2",datacenter="datacenter2",hostsystem="hostsystem2",statkey="number_of_vcpus_total",virtualmachine="vm1"}'
+      - 'vrops_vms_stats{cluster="cluster2",datacenter="datacenter2",hostsystem="hostsystem2",statkey="cpu_demand_ratio",virtualmachine="vm2"}'
```

Comment:
Please move the test for CollectorUp at the end of the file. Because your collectors won't be considered in the CollectorUp test. Then the test will fail, and these metrics will be marked as missing in the file. 
```
vrops_collector_up{collector="VMPropertiesCollector",metric="vrops_vm_properties"}
vrops_collector_up{collector="VMStatsCollector",metric="vrops_vms_stats"}
```

Reply:
This has been taken care, i just pushed the change. -Thanks